pkg/venus_test.go

```diff
@@ -0,0 +1,239 @@
+//go:build integration
+// +build integration
+
+package pkg
+
+import (
+	"context"
+	"math/big"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/require"
+)
+
+func TestVenusOperation_New(t *testing.T) {
+	t.Run("unsupported chain", func(t *testing.T) {
+		venus, err := NewVenusOperation(getTestClient(t, ChainETH),
+			big.NewInt(1),
+			common.HexToAddress("0xd933909A4a2b7A4638903028f44D1d38ce27c352"))
+		require.Error(t, err)
+		require.Nil(t, venus)
+	})
+
+	t.Run("venus correctly setup", func(t *testing.T) {
+		venus, err := NewVenusOperation(getTestClient(t, ChainBSC),
+			big.NewInt(56),
+			common.HexToAddress("0xd933909A4a2b7A4638903028f44D1d38ce27c352"))
+		require.NoError(t, err)
+		require.NotNil(t, venus)
+	})
+}
+
+func TestVenusOperation_Validate(t *testing.T) {
```

Comment:
Maybe we should discuss adding a Pause/Utilization/Health % API to avoid failed operations or funds loss.